src/api/operations/ce.queries.graphql

```diff
@@ -146,3 +146,16 @@ query GetAdminCeReferrals(
     }
   }
 }
+
+query GetUserCeAssignedSteps($id: ID!) {
+  user(id: $id) {
+    ceAssignedSteps {
```

Comment:
```suggestion
    id
    ceAssignedSteps {
```

I'm getting the following warning. I think this should resolve it, untested
<img width="783" alt="Screenshot 2025-05-23 at 3 17 59 PM" src="https://github.com/user-attachments/assets/6c4fccd0-21fd-4ec9-8e25-1d681849e7f9" />